### PR TITLE
Add function to edit and persistent the current options

### PR DESCRIPTION
**Pull request by MarcoChavez1940**
This PR is to: 
**#9 BugfIx: Edit option doesn't persist the data**

- Add function to edit and persistent the current options

### CODE REVIEW

File: src/components/Poll.vue [hunk 1]
```diff
@@ -74,6 +74,10 @@ export default {
       document.getElementById('add-option-input').focus()
     },
 
+    editOption(newOption) {
+      this.options = this.options.map((option) => { return option.id === newOption.id ? newOption : option })
```
Comment (rejonpardenilla) [inline src/components/Poll.vue]: if the arrow function is only one line you can skip the `return` statement and leave it like this:

`this.options.map(option => option.id === newOption.id ? newOption : option)`

Comment (MarcoChavez1940) [reply src/components/Poll.vue]: Nice, I like it.

**Comment by MarcoChavez1940** [comment]
> I like this solution but I think it can be improved a little more.
> 
> Also @Kami-Juan made another PR to solve the same problem in PR #10 . What do you think of it?

@rejonpardenilla I like the approach of @Kami-Juan of use a direct function what handle the local storage instead use a 'watch'. But his proposal of change the list of inputs and the 'updateOptionPersisted' function i think that add unnecessary complexity

**Comment by Kami-Juan** [comment]
> > I like this solution but I think it can be improved a little more.
> > Also @Kami-Juan made another PR to solve the same problem in PR #10 . What do you think of it?
> 
> @rejonpardenilla I like the approach of @Kami-Juan of use a direct function what handle the local storage instead use a 'watch'. But his proposal of change the list of inputs and the 'updateOptionPersisted' function i think that add unnecessary complexity

You right, i use the event input instead of the option value :grimacing:

**Pull request merged**